docs/source/packages/accounts-2fa.md

```diff
@@ -0,0 +1,241 @@
+      Meteor.loginWithPassword(username, password, error => {
+        if (error) {
+          console.error("Error trying to log in (user without 2fa)", error);
+        }
+      });
+    });
+  }
+}>
+  Login
+</button>
+```
+
+If the user has 2FA enabled, and you try to use the function `Meteor.loginWithPassword`, the login will fail, as the user should provide a code to access the app.
+
+The function you will need to call now to allow the user to login is `Meteor.loginWithPasswordAnd2faCode`:
+
+{% apibox "Meteor.loginWithPasswordAnd2faCode" %}
+
+Now you will be able to receive a code from the user and this function will verify if the code is valid. If it is, the user will be logged in.
+
+So the call of this function should look something like this:
+
+```js
+<button onClick={() => {
+  Meteor.loginWithPasswordAnd2faCode(username, password, code,error => {
+    if (error) {
+      console.error("Error trying to log in (user with 2fa)", error);
+    }
+  })}
+}>
+  Validate and log in
+</button>
+```
+
+<h4 id="working-with-accounts-passwordless">Working with accounts-passwordless</h4>
+
+Following the same strategy from the previous package, you can use the function `Accounts.has2faEnabled` to verify whether or not the user has 2FA enabled. If yes, you send them their token and on next step you receive their token and their 2FA code, otherwise, you still send them their token but on the next step you don't ask them for a 2FA code.
+
+Here it's an example:
+
+```js
+Accounts.has2faEnabled(username, (err, isEnabled) => {
+  if (err) {console.error("...", err);return;}
+  
+  Accounts.requestLoginTokenForUser({selector: "email@example.com"}, e => {
+     if (e) {console.error("...", e);return;}
+     
+     if (isEnabled) {
+      setShouldAskTokenAndCode(true);
+      return;
+     }
+
+    setShouldAskToken(true);
+  });
+});
+```
+
+Now you can either call the standard method [`Meteor.passwordlessLoginWithToken`](https://docs.meteor.com/packages/accounts-passwordless.html#Meteor-passwordlessLoginWithToken) if they don't have 2FA enabled, or in case they do, you call the method `Meteor.passwordlessLoginWithTokenAnd2faCode` that will allow you to provide a selector, token, and 2FA code:
+
+{% apibox "Meteor.passwordlessLoginWithTokenAnd2faCode" %}
+
+So, using this strategy your code should look something like this:
+
+```js
+<button
+  onClick={() => {
+    // logging in with token and code
+    if (shouldAskTokenAndCode) {
+      Meteor.passwordlessLoginWithTokenAnd2faCode(
+        email,
+        token,
+        code,
+        error => {
+          if (error) {console.error('...', error);}
+        }
+      );
+      return;
+    }
+    // logging in just with token 
+    Meteor.passwordlessLoginWithToken(
+      email,
+      token,
+      error => {
+        if (error) {console.error('...', error);}
+      }
+    );
+  }}
+>
+  Validate and Log in
+</button>;
+```
+
+<h3 id="disabling-2fa">Disabling 2FA</h3>
```

Comment:
This should be before the examples with passwordless and password.

Also, these examples should be here or in their own packages?

At least their doc should mention this doc.